packages/InputText/styles.ts

```diff
@@ -13,18 +13,21 @@ export const InputText = styled('input').withConfig({ shouldForwardProp })<Input
     ${icon &&
     iconPlacement === 'left' &&
     css`
-      padding-left: ${th(`defaultFields.sizes.${size}.height`)};
+      padding-left: 36;
     `};
-    ${icon &&
-    iconPlacement === 'right' &&
+
+    /* With clear button or right icon */
+    ${(isClearable || (icon && iconPlacement === 'right')) &&
     css`
-      padding-right: ${th(`defaultFields.sizes.${size}.height`)};
+      padding-right: 36;
```

Comment:
For line 16 and line 22, padding-right value shouldn't be theme values?

Reply:
As padding defines the distance between the input value and the icon, it's ok to keep 36 as padding value